src/rules/indentRule.ts

```diff
@@ -121,6 +128,161 @@ function walk(ctx: Lint.WalkContext<Options>): void {
     }
 }
 
+function checkIndentSize(params: {
+    ctx: Lint.WalkContext<Options>;
+    tabs: boolean;
+    size: number;
+    pos: number;
+    indentEnd: number;
+    line: string;
+    sourceFile: ts.SourceFile;
+}): boolean {
```

Comment:
consider moving this function inside `walk` so it closes over the variables like `ctx`, `tabs`, `sourceFile` and so on.

Reply:
done